docs/docs/PluginDevelopment.md

```diff
@@ -21,24 +20,31 @@ For example:
 ```bash
     dvp build
 ```

```

Comment:
It looks like all these lines are rendered in one code block.

Reply:
Fixed.